code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -188,3 +188,10 @@
 /obj/item/reagent_containers/hypospray/autoinjector/nanocalcium/attack(mob/living/M, mob/user)
 	if(..())
 		playsound(loc, 'sound/weapons/smg_empty_alarm.ogg', 20, 1)
+
+/obj/item/reagent_containers/hypospray/autoinjector/magillitis
+	name = "experimental autoinjector"
+	desc = "A custom-frame needle injector with a small single-use reservoir, containing an experimental serum. Unlike the more common medipen frame, it cannot pierce through protective armor or space suits, nor can the chemical inside be extracted."
+	icon_state = "gorillapen"
```

Comment:
this doesn't exist in your current branch, causing the injector to be invisible in game

Reply:
Thanks, i deleted this to fix the issue.